rocAL/rocAL_pybind/amd/rocal/fn.py

```diff
@@ -283,6 +283,30 @@ def resize(*inputs, bytes_per_sample_hint=0, image_type=0, interp_type=1, mag_fi
     resized_image = b.Resize(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
     return (resized_image)
 
+def resize_mirror_normalize(*inputs, bytes_per_sample_hint=0, resize_min=0, resize_max=0, 
+                            image_type=0, mean=[0.0], mirror=1, output_dtype=types.FLOAT, output_layout=types.NCHW, pad_output=False,
+                            preserve=False, seed=1, std=[1.0], device=None):
+    
+    #Set Seed
+    b.setSeed(seed)
```

Comment:
Remove both the setSeed() instances from here.
There are 2 instances.
Set the seed in the pipeline.py itself.

Reply:
Resolved in the latest commit